--- localsettings.example.py	2018-09-11 14:23:19.000000000 -0700
+++ localsettings.py	2018-09-13 09:27:10.000000000 -0700
@@ -112,7 +112,10 @@
 
 ####### Log/debug setup ########
 
-DEBUG = True
+DEBUG = False
+ALLOWED_HOSTS = [
+    '.auderenow.io',
+]
 
 # log directories must exist and be writeable!
 DJANGO_LOG_FILE = "/tmp/commcare-hq.django.log"
@@ -124,9 +127,9 @@
 
 ####### Less/Django Compressor ########
 
-LESS_DEBUG = True
-COMPRESS_ENABLED = False
-COMPRESS_OFFLINE = False
+LESS_DEBUG = False
+COMPRESS_ENABLED = True
+COMPRESS_OFFLINE = True
 
 ####### Bitly ########
 
@@ -213,7 +216,7 @@
 #    'testapps.test_elasticsearch',
 #    'testapps.test_pillowtop',
 #    'django_fsm', # Adds the ability to generate state diagrams for models using django-fsm
-#    'kombu.transport.django', # required for celery
+    'kombu.transport.django', # required for celery
 #    'package_monitor',
 )
 
@@ -316,3 +319,5 @@
 # The passcodes will print out on the console
 # TWO_FACTOR_CALL_GATEWAY = 'two_factor.gateways.fake.Fake'
 # TWO_FACTOR_SMS_GATEWAY = 'two_factor.gateways.fake.Fake'
+
+FORMPLAYER_URL = 'http://localhost:8010'
